cdap-ui/app/cdap/text/text-en.yaml

```diff
@@ -823,7 +823,7 @@ features:
         title: Configure a Pipeline
         description: Specify the name of the pipeline.
       pipelinenameplaceholder: Pipeline Name
-      headerlabel: Publish Pipeline
+      headerlabel: Deploy Pipeline
```

Comment:
Should it say `Create` instead? This is just creating a draft right? It's not deployed yet